python2/raygun4py/raygunmsgs.py

```diff
@@ -90,8 +90,10 @@ def set_customdata(self, user_custom_data):
         return self
 
     def set_tags(self, tags):
-        if type(tags) is list or tuple:
-            self.raygunMessage.details['tags'] = tags
+        if type(tags) is list:
```

Comment:
We are not checking for tuples here anymore.
Is that a breaking change, or can tuples be coerced in to a list in python?